From 1c7522d9898cf623e34998de121e8d7c4cebc215 Mon Sep 17 00:00:00 2001
From: Ted Ross <tross@apache.org>
Date: Wed, 9 Jun 2010 15:16:22 +0000
Subject: [PATCH 06/11] Bug 602313 - qpid-tool does not show correct "deleted" timestamp on deleted objects

Update the current and deleted timestamps in the "mergeUpdate" method.

git-svn-id: https://svn.apache.org/repos/asf/qpid/trunk@953039 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 1f619960d27ba117e34149722e4a9f7702bededc)
---
 qpid/extras/qmf/src/py/qmf/console.py |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/qpid/extras/qmf/src/py/qmf/console.py b/qpid/extras/qmf/src/py/qmf/console.py
index 6cfcdc9..02c5ff1 100644
--- a/qpid/extras/qmf/src/py/qmf/console.py
+++ b/qpid/extras/qmf/src/py/qmf/console.py
@@ -279,6 +279,8 @@ class Object(object):
       self._properties = newer.getProperties()
     if len(newer.getStatistics()) > 0:
       self._statistics = newer.getStatistics()
+    self._currentTime = newer._currentTime
+    self._deleteTime = newer._deleteTime
 
   def update(self):
     """ Contact the agent and retrieve the lastest property and statistic values for this object. """
-- 
1.7.0.1

